src/dune_lang/relop.mli

```diff
@@ -0,0 +1,15 @@
+type t =
+  | Eq
```

Comment:
We could even make this a polymorphic variant and avoid yet another set of conversions.